### DISCUSSION TITLE
Disable Fan Control and Monitoring

### DISCUSSION THREAD

**Discussion by metalgie** [Rem0o/FanControl.Releases#791] [Ideas]
I want to start with a big thanks for your amazing software, I love it and it is extremely hard to live without, I want to ask for a feature to be added, I have lian li al120 fans. The fans cable is connected directly to my motherboard, the fans lowest speed is around 800rpm, when the MB control those fans, I can set the speed at the lowest, but when FanControl start, the fan lowest speed gose up to around 920-940rpm, Fancontrol can't detect fan start % or fan stop %, is there a way I can tell Fancontrol to never take control of a specific fan?

**Comment by Rem0o** [comment]
Leave the control card disabled:
![image](https://user-images.githubusercontent.com/14118956/168617101-1c5fb104-3220-4331-a335-9625958de76c.png)
